apps/charterafrica/src/lib/data/common/index.js

```diff
@@ -17,25 +17,38 @@ import processPagePrivacyPolicy from "@/charterafrica/lib/data/common/processPag
 import processPageTools from "@/charterafrica/lib/data/common/processPageTools";
 import { getPageSeoFromMeta } from "@/charterafrica/lib/data/seo";
 
+function getLogoProps(logo) {
+  const {
+    href = "/",
+    image: {
+      alt = "Charter Africa",
+      fill = true,
+      priority = true,
+      src = "/images/charter-logo.svg",
```

Comment:
```suggestion
      url:src = "/images/charter-logo.svg",
```

Reply:
What is url:src?

Reply:
Changing the logo in cms does not update the image logo on the front page as `src=/images/charter-logo.svg`. The payload from the CMS has the image source as `url`.
So we need to rename the `url` to `src` and then default it to `/images/charter-logo.svg`
Here is a sample payload: 
```
image: {
      id: '641a034566df56a12ded82ae',
      alt: 'People',
      prefix: 'media',
      filename: 'people-icon.svg',
      mimeType: 'image/svg+xml',
      filesize: 1008,
      width: 125,
      height: 124,
      createdAt: '2023-03-21T19:19:33.269Z',
      updatedAt: '2023-03-21T19:19:33.269Z',
      url: 'http://localhost:3000/media/people-icon.svg'
    },
   linkType: 'internal',
    doc: { value: [Object], relationTo: 'pages' },
    href: '/opportunities/helpdesk'
 }
```

Reply:
Oh... good catch but you can't change that via `url:src? = `.